ckan/pastertemplates/template/setup.py_tmpl

```diff
@@ -54,6 +54,7 @@ setup(
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+    namespace_packages=['ckanext', 'ckanext.{{ project_shortname }}'],
```

Comment:
Yes, `ckanext` should be a namespace package, but `ckanext.{{ project_shortname }}` should not.